### DISCUSSION TITLE
Is there any way I can validate only one authenticator device allow to register one time in RP?

### DISCUSSION THREAD

**Discussion by Sovannak22** [Yubico/java-webauthn-server#362] [Q&A] [answered]
Hello I want to implement logic to handle one device can only register one time in my RP server. Since device like YubiKey can register multiple time in one RP with difference public key and  credential id is there any information that unique per device that I can use for this validation? Thank.

**Comment by emlun** [comment, marked as answer]
Hi! This sounds like a duplicate of issue #354. Does that thread answer your question?

In short, the answer depends on whether you want to prevent registering the same authenticator to the _same account_ more than once (this is possible, and the library does this automatically) or prevent registering the same authenticator to _more than one account_ (this is impossible by the design of WebAuthn).

**Reply by Sovannak22** [reply]
Oh yes thank you it's answer my question. I want to prevent registering the same authenticator to more than one account in one RP. Since this is not possible I have to find another way.

**Reply by emlun** [reply]
Just out of curiosity: why do you want to prevent that?
